css/style.css

```diff
@@ -72,4 +72,100 @@ li{
     -ms-border-radius: 50%;
     -o-border-radius: 50%;
 }
-/***** End general *****/
\ No newline at end of file
+/***** End general *****/
+/***** hero *****/
+.hero{
+    height: 100vh;
+    background: linear-gradient(
+        rgba(0,0,0,.3), 
+        rgba(0,0,0,.3)),url(../assets/images/hero-img.jpeg) no-repeat 50%;
+    background-size: cover;
+    position: relative;
+    z-index: 1;
+}
+.hero .flexContainer{
+    flex-direction: column;
+    height: 100%;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+}
+.heroHeader{
+    color: var(--white-text);
+    text-align: center;
+}
+.heroHeader h1{
+    font-size: 4.375rem;
+    margin-bottom: 1.875rem;
+}
+.heroHeader h4, .heroHeader p{
+    font-size: 1.5rem;
+    margin-bottom: 2.5rem;
+    color: #ccc;
+    line-height: 1.86em;
+}
+.active{
+    border: none;
+    background: var(--primary-bg);
+    margin-right: 0.625rem;
+}
+.disabled{
+    border: 2px solid var(--white-text);
+    background: none;
+}
+.active:hover, .disabled:hover{
+    background-color: var(--white-text);
+    color: var(--dark-text);
+    transition: all .3s;
+    -webkit-transition: all .3s;
+    -moz-transition: all .3s;
+    -ms-transition: all .3s;
+    -o-transition: all .3s;
+}
+.scroll{
```

Comment:
animation internal dash is not centred
should be :
![image](https://user-images.githubusercontent.com/44810632/155527039-b9619155-e858-4206-942e-f642fa6e0add.png)
but it is:
![image](https://user-images.githubusercontent.com/44810632/155527069-82b06285-a074-41db-84b4-a0a1a7f12246.png)